### PR TITLE
CSIT-896 Create test for regression bug P2PS-2560

### PR DESCRIPTION
**Pull request by Alena-deriv**
ClickUp card https://app.clickup.com/t/20696747/CSIT-896

Description:
1. Test creates 3 Payments methods from BO side
2. Test checks if Payment methods are already exist, if yes, will proceed with further steps, if no - will create new ones.
3. On the DOUGHFLOW PAYMENT METHODS page, test will sort all available payment methods by 'Category' and will check that result with 'Category1' is filtered
Screenshot 2024-06-25 at 16 03 42

### CODE REVIEW

**Review by bilalmajid-deriv — APPROVED** [review]

**Comment by bilalmajid-deriv** [comment]
Approved the usage of cy.get as it is tricky sometimes not possible to find a testing library locator

File: cypress/e2e/smoke/CSIT/boPaymentMethodSorting.cy.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    if ($el.text().includes('Category3')) {
+      cy.log('PM3 category exists')
+    } else {
+      checkAndCreatePM3()
+      cy.log('Added PM3 category')
+    }
+  })
+}
+
+describe('QATEST-140205 - Sorting using category in BO Doughflow payment methods management page', () => {
```
Comment (meeyun-deriv) [inline cypress/e2e/smoke/CSIT/boPaymentMethodSorting.cy.js:72]: this ID is reference to test case. We need to reference to the test scenarios instead

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/smoke/CSIT/boPaymentMethodSorting.cy.js (hunk 1 above)
Comment (Alena-deriv) [reply cypress/e2e/smoke/CSIT/boPaymentMethodSorting.cy.js]: Updated

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**